### ISSUE TITLE
Clarification on Block executables from running until they meet criteria

### ISSUE THREAD

**Issue by the123blackjack** [MicrosoftDocs/windows-itpro-docs#945]
I have enabled this asr (Block executables from running until they meet age, prevalence or trusted criteria) it seems to be not taking effect. I am able to run all executables. Also where do i set the " age prevalence and trusted criteria?)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f755cdc0-8b2e-fc10-29de-3ee9a023bd02
* Version Independent ID: 9b5f713b-4e4e-a159-8d90-aa31b4dbb0cc
* Content: [Configure how ASR works to finetune protection in your network](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/customize-attack-surface-reduction)
* Content Source: [windows/security/threat-protection/windows-defender-exploit-guard/customize-attack-surface-reduction.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-defender-exploit-guard/customize-attack-surface-reduction.md)
* Service: **unspecified**
* Product: **w10**
* GitHub Login: @andreabichsel
* Microsoft Alias: **v-anbic**

**Comment by the123blackjack** [comment]
I have been using asr on Windows pro ... It's only for enterprise ?

**Comment by andreabichsel** [comment]
Yes. I don't think the requirements were clear before, so I recently updated Windows Defender Exploit Guard topics with the versions each feature is supported in.
